browser/ui/webui/playlist_ui.h

```diff
@@ -77,18 +78,16 @@ class PlaylistUI : public ui::UntrustedWebUIController,
   WEB_UI_CONTROLLER_TYPE_DECL();
 };
 
-class UntrustedPlaylistUIConfig : public content::WebUIConfig {
+class UntrustedPlaylistUIConfig
+    : public DefaultTopChromeWebUIConfig<PlaylistUI> {
  public:
   UntrustedPlaylistUIConfig();
   ~UntrustedPlaylistUIConfig() override = default;
 
-  std::unique_ptr<content::WebUIController> CreateWebUIController(
-      content::WebUI* web_ui,
-      const GURL& url) override;
-
   bool IsWebUIEnabled(content::BrowserContext* browser_context) override;
 };
 
+// TODO (Sangwoo): Convert to subclassing DefaultTopChromeWebUIConfig?
```

Comment:
cc: @sangwoo108, wasn't sure about this one, especially since the class is hidden in the .cc

Reply:
I think it'd be nice to convert it and remove CreateWebUIController override.

Reply:
I am not sure there's a way to do this without bringing the declaration of `class UntrustedPlayerUI` into the header file. Is there a particular reason the class is declared/defined in the anonymous namespace in the .cc file? Are there security implications?

Reply:
Is this player used for "Brave Player" feature? If so, it doesn't seem like it uses WebUIContentsWrapperT, in  which case we don't need to use `DefaultTopChromeWebUIConfig`.

Reply:
> Is this player used for "Brave Player" feature?

No it's not. It's a subframe of Playlist web UI. That's partly why I put this in the anonymous namespace. But it seems okay to move it to header.